source/Generic/PlayState/PlayStateSettings.cs

```diff
@@ -56,12 +64,26 @@ public PlayStateSettingsViewModel(PlayState plugin)
             if (savedSettings != null)
             {
                 Settings = savedSettings;
+                bool modifiedSettings = false;
                 if (Settings.SavedHotkeyGesture.Modifiers == ModifierKeys.None)
                 {
                     // Due to a bug in previous version that allowed 
                     // to save gestures without modifiers, this
                     // should be done to restore the default ModifierKeys
                     Settings.SavedHotkeyGesture.Modifiers = (ModifierKeys)5;
+                    modifiedSettings = true;
+                    
+                }
+                if (Settings.SavedInformationHotkeyGesture.Modifiers == ModifierKeys.None)
+                {
+                    // Due to a bug in previous version that allowed 
+                    // to save gestures without modifiers, this
+                    // should be done to restore the default ModifierKeys
+                    Settings.SavedInformationHotkeyGesture.Modifiers = (ModifierKeys)5;
+                    modifiedSettings = true;
+                }
```

Comment:
Why is this being done for the new hotkeys? The bug should not exist there right?

Reply:
Like I said before I don't understand well the logic behind hotkeys. I thought this was executed after editing hotkeys all the time. I will remove it then.